main.go

```diff
@@ -34,16 +36,37 @@ func isFlagPassed(name string) bool {
 	return found
 }
 
-var version = "dev"
+var (
+	version  = "dev"
+	logLevel string
+)
 
-func main() {
+func init() {
 	flag.BoolP("version", "v", false, "show version")
+	flag.StringVarP(&logLevel, "log-level", "L", "info", "Log level (debug, info, warn, error, fatal, panic)")
```

Comment:
Do we need all of these different debug levels? I think having a flag for debugging is a great idea (I wish I had this when I was originally writing the code for loading the config files!!), I think we maybe only need to be able to toggle debugging on or off, which could be achieved like `meteor --debug` or `meteor -D`, as `git commit` does not use the `--debug` or `-D` flags

Reply:
let me update that then. hold on.

Reply:
Added some changes:
<img width="1527" alt="image" src="https://github.com/stefanlogue/meteor/assets/2515929/4936657e-8f2d-49e5-909d-465e8ed55390">

Reply:
```
❯ go run main.go git.go config.go --help
Usage of /var/folders/yx/7rrwmnvd3d14_qc362y5yt300000gn/T/go-build713247778/b001/exe/main:
  -D, --debug     enable debug mode
  -v, --version   show version
pflag: help requested
exit status 2
```

Reply:
About cobra, let's discuss this in a new PR or issue.

Reply:
> Added some changes: <img alt="image" width="1527" src="https://private-user-images.githubusercontent.com/2515929/314526052-4936657e-8f2d-49e5-909d-465e8ed55390.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NDc4MzQsIm5iZiI6MTcxMDk0NzUzNCwicGF0aCI6Ii8yNTE1OTI5LzMxNDUyNjA1Mi00OTM2NjU3ZS04ZjJkLTQ5ZTUtOTA5ZC00NjVlOGVkNTUzOTAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjBUMTUxMjE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjQwM2YzZDE0MTc3ODE4YmE2YjRmYTlkNzNlZWU1ZDU1ZDE4YTQxNDZmOGY5ZDc1ZGY5NzFhZmU0MTFmOWE4ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.zje_r_sLyM8wGWcr0P_rSZqEx7M2y2qfNAB8ArHMl8c">

This looks good, but I'm wondering if we want to raise a warning when a user has no config file? I imagine the vast majority of users won't need a config file at all if they just want to do vanilla conventional commits

Reply:
ok, let's move to debug then.

Reply:
807b62578ad1ed027865ad910586b1651155c759 should fix the problem.